src/domain/resource/card/__tests__/__snapshots__/ResourceCard.test.js.snap

```diff
@@ -105,6 +106,43 @@ exports[`ResourceCard renders correctly 1`] = `
       icon="test-file-stub"
       text="null"
     />
+    <span
+      className="app-resourceCardInfoCell"
+    >
+      <span
+        className="app-ResourceCardInfoCell__loginMethods"
+      >
+        <img
```

Comment:
The string authentication methods should probably also be added here.

Reply:
This file is autogenerated by jest , its a snapshot. So I think its okay as it is ?